composer.json

```diff
@@ -31,12 +31,12 @@
         "symfony/monolog-bundle": "For logging messages"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.0",
+        "phpunit/phpunit": "^5.7",
         "simple-bus/doctrine-orm-bridge": "~5.0",
         "doctrine/orm": "~2.3",
         "doctrine/doctrine-bundle": "~1.0",
-        "symfony/monolog-bridge": "~2.3 || ~3.0",
-        "symfony/monolog-bundle": "~2.3 || ~3.0"
+        "symfony/monolog-bridge": "~2.7 || ~3.0 || ~4.0",
+        "symfony/monolog-bundle": "~2.7 || ~3.0 || ~4.0"
```

Comment:
the release cycle of MonologBundle is independent from Symfony's release cycle (see https://github.com/symfony/monolog-bundle/releases)

Reply:
Ops, true.